src/it/resources/org/checkstyle/suppressionxpathfilter/methodtypeparametername/InputXpathMethodTypeParameterName1.java

```diff
@@ -1,5 +1,5 @@
 package org.checkstyle.suppressionxpathfilter.methodtypeparametername;
 
-public class SuppressionXpathRegressionMethodTypeParameterName1{
+public class InputXpathMethodTypeParameterName1{
```

Comment:
Is it possible to find a better suffix than "1" ?
Please try to find a proper name for `InputXpathMethodTypeParameterName1` --> `InputXpathMethodTypeParameterNameXxxx`
Same for all others....

Reply:
You make sense. done